README_new.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+At babylon, we firmly believe that *transparency* and *flexibility* are core values that should be present in everything we do, and hiring the best person for a position should reflect that belief.
```

Comment:
~for a position~

Reply:
Fixed in https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/0d0fea2d42af712eae83df7ca4aca779f0e923d2.